docs/spec/glossary.rst

```diff
@@ -12,15 +12,77 @@ This section defines a few terms that may be used elsewhere in the specification
       :term:`type expression`, sometimes with additional :term:`type qualifiers <type qualifier>`.
       See :ref:`"Type and annotation expression" <annotation-expression>` for details.
 
+   assignable
+      If a type ``B`` is "assignable to" a type ``A``, a  type checker should
+      not error on the assignment ``x: A = b``, where ``b`` is some expression
+      whose type is ``B``. Similarly for function calls and returns: ``f(b)``
+      where ``def f(x: A): ...`` and ``return b`` inside ``def f(...) -> A:``
+      are both valid (not a type error) if and only if ``B`` is assignable to
+      ``A``. In this case ``A`` is "assignable from" ``B``. For :term:`fully
+      static types <fully static type>`, "assignable to" is equivalent to
+      ":term:`subtype` of" and "assignable from" is equivalent to
+      ":term:`supertype` of". For :term:`gradual types <gradual type>`, a type
+      ``B`` is assignable to a type ``A`` if there exist :term:`fully static
+      <fully static type>` :term:`materializations <materialize>` ``A'`` and
+      ``B'`` of ``A`` and ``B``, respectively, such that ``B'`` is a
+      :term:`subtype` of ``A'``. See :ref:`type-system-concepts`.
```

Comment:
Too many links in the same paragraph can be distracting for the reader. In the CPython docs, the rule we usually use is to not link to the same thing more than once in the same paragraph. The glossary definitions for "fully static" and "subtype" were both linked to earlier on in the paragraph:

```suggestion
      ``B`` is assignable to a type ``A`` if there exist :term:`fully static
      :term:`materializations <materialize>` ``A'`` and ``B'`` of ``A`` and
      ``B``, respectively, such that ``B'`` is a subtype of ``A'``. See
      :ref:`type-system-concepts`.
```

There's some similar issues in some of the other entries you've added here